test/constants.ts

```diff
@@ -19,8 +19,6 @@ export const actionSelectors = {
 
 export const widgetLocation = {
   entryField: 'entry-field',
-  /** @deprecated */
-  entryFieldSidebar: 'entry-field-sidebar',
```

Comment:
this is deprecated simply because this concept was specific to legacy UI extensions, right? (field editor *as a sidebar extension*?)

Reply:
Yes. entry field sidebar is extensions only